db/models.py

```diff
@@ -42,11 +51,102 @@ def __str__(self) -> str:
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
     cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=CinemaHall, on_delete=models.CASCADE, related_name="cinema_hall"
     )
     movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=Movie, on_delete=models.CASCADE, related_name="movie"
     )
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at.strftime('%Y-%m-%d %H:%M:%S')}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        "MovieSession",
+        on_delete=models.CASCADE,
+        null=False
+    )
+    order = models.ForeignKey(
+        "Order",
+        on_delete=models.CASCADE,
+        null=False
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    constraints = [
+        UniqueConstraint(
+            fields=["row", "seat", "movie_session"],
+            name="unique_ticket")]
+
+    def clean(self) -> None:
+        existing_tickets = Ticket.objects.filter(
+            row=self.row,
+            seat=self.seat,
+            movie_session=self.movie_session
+        )
+
+        if existing_tickets.exists():
+            raise ValidationError("Not Unique Value")
+
+        if not (
+                self.row <= self.movie_session.cinema_hall.rows
+        ):
```

Comment:
There is no need to complicate this check with `not`, you could omit it
```suggestion
        if self.row > self.movie_session.cinema_hall.rows:
```